README.md

```diff
@@ -1 +1,2 @@
-# review-my-trial
\ No newline at end of file
+# Lynda-Yared-Microverse-Trial
+Lynda-Yared
```

Comment:
I suggest filling your README.md file so it looks professional and has the information to be shown to your visitors.